{
    "directory name": "llvm-project/libcxx/test/std/language.support/support.exception/bad.exception",
    "score.sum": 0.000903130118213133,
    "sha.latest": "7fc6a55688c816f5fc1a5481ae7af25be7500356",
    "time.latest": 1717922767.3040428,
    "formatted_time": "2024-06-09 14:16:07.304044",
    "filescores": {
        "bad_exception.pass.cpp": 0.057160031765472394
    },
    "subdirscores": {}
}